tests/internal/test_settings.py

```diff
@@ -235,6 +254,29 @@ def test_remoteconfig_custom_tags(run_python_code_in_subprocess):
     assert status == 0, f"err={err.decode('utf-8')} out={out.decode('utf-8')}"
 
 
+def test_remoteconfig_tracing_enabled(run_python_code_in_subprocess):
+    env = os.environ.copy()
+    env.update({"DD_TRACE_ENABLED": "true"})
+    out, err, status, _ = run_python_code_in_subprocess(
+        """
+from ddtrace import config, tracer
+from tests.internal.test_settings import _base_rc_config
+
+assert tracer.enabled is True
+
+config._handle_remoteconfig(_base_rc_config({"tracing_enabled": "false"}))
+
+assert tracer.enabled is False
+
+config._handle_remoteconfig(_base_rc_config({"tracing_enabled": "true"}))
+
+assert tracer.enabled is False
```

Comment:
Are you confident that this signal corresponds strong enough to no traces being sent?

Reply:
Yes, mainly because of [this test](https://github.com/DataDog/dd-trace-py/blob/main/tests/commands/ddtrace_run_disabled.py).